.github/workflows/e2e-upgrade-test-reusable.yaml

```
@@ -0,0 +1,110 @@
+name: Upgrade Tests Workflow (reusable)
+
+env:
+  E2E_LOG_LEVEL: debug
+  KYMA_STABILITY: "unstable"
+  KYMA: "./hack/kyma"
+
+on:
+  workflow_call:
+    inputs:
+      # image tag before upgrade
+      pre-upgrade-image-tag:
+        required: true
+        type: string
```

When I asked for a `description` I had this in my mind: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_dispatchinputs

```suggestion
      pre-upgrade-image-tag:
        required: true
        type: string
        description: The tag of the image used before the upgrade.
        ```